.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules/
+/.idea/
```

Comment:
No newline at end of file.

Btw. chciałem, żebyś wygenerował .gitignore za pomocą PHPStorm.

Tutaj jest poprawny plik .gitignore dla NodeJS:
https://github.com/pawelzny/element-strategy/blob/master/.gitignore